techmatters/flex-plugins
# Update Flex UI Version Automatically on Deploy

Primary reviewer: @GPaoloni 

## Description
This PR adds one more step to the deployment flow:
- It now sets **Flex UI Version** on the deployed environment to be the same as in the `package-lock.json` file

### Verification steps
If needed, you can manually try this on development:
1. Manually downgrade UI Version in here: https://flex.twilio.com/admin/versioning/
2. Trigger deploy from this branch
3. Refresh the Flex Page and see the UI Version has changed
